apps/etterlatte-saksbehandling-ui/client/src/components/behandling/aktivitetsplikt/Aktivitetsplikt.tsx

```diff
@@ -69,7 +73,7 @@ export const Aktivitetsplikt = (props: { behandling: IDetaljertBehandling }) =>
             Oppfølging av aktivitet
           </Heading>
           <BodyShort spacing>
-            <strong>Dødsdato: </strong> {avdoedesDoedsdato}
+            <strong>Dødsdato: </strong> {avdoedesDoedsdato && formaterDato(new Date(avdoedesDoedsdato))}
```

Comment:
Burde man hatt en melding her om dødsdato ikke finnes?

Reply:
Så vidt jeg vet skal det i praksis aldri skje. Mest for å slippe `!!`.
Kan undersøke litt mens jeg holder på og legge det til hvis det er tilfelle.